python/scisql/configure.py

```diff
@@ -92,7 +91,7 @@ def apply_tpl(src_file, target_file):
     out_cfg = t.safe_substitute(**params_dict)
     for match in t.pattern.findall(t.template):
```

Comment:
@smonkewitz My IDE is complaining about this. Is there some sort of magic that happens in ScisqlConfigTemplate which I'm unaware of which makes this okay? t.pattern isn't defined as a regex but it has the `r""` string below in this file.